lib/active_model/serializer.rb

```diff
@@ -111,6 +108,200 @@ def self.serialization_adapter_instance
       @serialization_adapter_instance ||= ActiveModelSerializers::Adapter::Attributes
     end
 
+    # Configuration options may also be set in
+    # Serializers and Adapters
+    config.collection_serializer = ActiveModel::Serializer::CollectionSerializer
+    config.serializer_lookup_enabled = true
+
+    # @deprecated Use {#config.collection_serializer=} instead of this. Is
+    #   compatibilty layer for ArraySerializer.
+    def config.array_serializer=(collection_serializer)
+      self.collection_serializer = collection_serializer
+    end
+
+    # @deprecated Use {#config.collection_serializer} instead of this. Is
+    #   compatibilty layer for ArraySerializer.
+    def config.array_serializer
+      collection_serializer
+    end
+
+    config.default_includes = '*'
```

Comment:
Noting needed documentation: 

- https://github.com/rails-api/active_model_serializers/pulls?q=is%3Aopen+is%3Apr+label%3A%22C%3A+Deep+Nesting%22 and https://github.com/rails-api/active_model_serializers/pulls?q=is%3Aopen+is%3Aissue+label%3A%22C%3A+Deep+Nesting%22